rspec-cells.gemspec

```diff
@@ -7,10 +7,19 @@ Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.authors     = ["Nick Sutterer"]
   s.email       = ["apotonick@gmail.com"]
-  s.homepage    = "http://rubygems.org/gems/rspec-cells"
+  s.homepage    = "https://rubygems.org/gems/rspec-cells/versions/#{s.version}"
   s.summary     = %q{Spec your cells.}
   s.description = %q{Use render_cell in your specs.}
   s.license     = 'MIT'
+
+  s.metadata = {
+    "bug_tracker_uri"   => "https://github.com/trailblazer/rspec-cells/issues",
+    "changelog_uri"     => "https://github.com/trailblazer/rspec-cells/blob/v#{s.version}/CHANGES.md",
+    "documentation_uri" => "https://www.rubydoc.info/gems/rspec-cells/#{s.version}",
+    "source_code_uri"   => "https://github.com/trailblazer/rspec-cells/tree/v#{s.version}",
```

Comment:
remove version

Reply:
The metadata is associated to each release. ie.

- version 0.3.6 will get https://github.com/trailblazer/rspec-cells/tree/v0.3.6
- version 0.4.0 will get https://github.com/trailblazer/rspec-cells/tree/v0.4.0

Pointing to the specific code included in the release can be quite helpful when debugging or investigating.

I curious to know the reasons it shouldn't include the version. But I'm happy to remove it.

Reply:
Sorry, i meant it for changelog_uri. 

Changelog is global.

Reply:
👍 Updated in a94e4cfc2168aa4aeaca04289de3c5d8340b24d2